ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,147 @@
+---
+eip: xxxx
+title: Sustainable Collections
+description: A standard for managing NFTs with dynamic fees and donation-based engagement.
+author: Gustavo Lobo (@gflobo)
+discussions-to: <URL>
```

Comment:
Please create a discussions topic on Eth Magicians: https://ethereum-magicians.org/c/ercs/57
It just needs to be a link to this PR.

Reply:
I apparently can't create discussions there yet because i have no level in the community enough.